datafusion/common/src/config.rs

```diff
@@ -180,6 +184,23 @@ macro_rules! config_namespace {
     }
 }
 
+config_namespace! {
+    pub struct ConfigFileDecryptionProperties {
+        pub footer_key: String, default = String::new()
+        pub column_keys: String, default = String::new()
+        pub aad_prefix: String, default = String::new()
+    }
+}
+
+config_namespace! {
+    pub struct ConfigFileEncryptionProperties {
+        pub encrypt_footer: bool, default = false
+        pub footer_key: String, default = String::new()
+        pub column_keys: String, default = String::new()
+        pub aad_prefix: String, default = String::new()
+    }
+}
+
```

Comment:
For now, create primitive Config structures for encryption/decryption that we copy back and forth from.